lighthouse-core/test/audits/metrics/cumulative-long-queuing-delay-test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/metrics/cumulative-long-queuing-delay.js');
```

Comment:
can you name this `cumulativeLongQueuingDelay` or `cLQDAudit` or whatever else just so long as it's not plain old `Audit` :) Helps the test cases to give a bit more inherent context